Lively/src/util.tsx

```diff
@@ -0,0 +1,24 @@
+export const validTeamName = (name: string) => {
```

Comment:
query functions should begin with a predicate, like isTeamNameValid. to take it a step further, try to think of how it will look with named arguments. do you see the problem with the name I proposed after thinking about the argument it will take? hint: there is a better answer